docs/providers/aws/guide/credentials.md

```diff
@@ -28,6 +28,19 @@ This guide is for the Amazon Web Services (AWS) provider, so we'll step through
 
 The Serverless Framework provides multiple methods to connect to AWS. However, the recommended configuration can be time consuming. Below we describe a quick way to get Serverless Framework connected to AWS securely. After that we provide the preferred configuration using multiple accounts to manage production and development deployments of your Serverless application.
 
+- [Quick Connect using Serverless Provider](#quick-connect)
+- [Using Locally Stored Credentials](#using-locally-stored-credentials)
+- [Advanced Credential Management (AWS SSO, Profiles, Roles & More)](#advanced-credential-management-aws-sso-profiles-roles--more)
+- [Using Resolvers to Specify Deployment Credentials](#using-resolvers-to-specify-deployment-credentials)
+  - [Defining Resolvers](#defining-resolvers)
+  - [Using Multiple Resolvers](#using-multiple-resolvers)
+  - [Using Serverless Dashboard Providers with Resolvers](#using-serverless-dashboard-providers-with-resolvers)
+- [Production Configuration](#production-configuration)
+  - [Management account](#management-account)
+  - [Stage accounts for production and development](#stage-accounts-for-production-and-development)
+  - [Developer accounts](#developer-accounts)
+  - [Combine with Serverless Provider](#combine-with-serverless-provider)
+
 ## Quick Connect
```

Comment:
Currently the "Quick Connect using Serverless Provider" is the first/recommended approach. However, I think we should change this a bit.

- Using the "Local Stored Credentials" setup with `serverless` should be the recommend approach first, so please rearrange this such that this is first/recommended. I'd also consider changing the title to something like "Recommended: Using Local Credentials".
- The "Quick Connect using Serverless Provider" should be another option (but not the first). As such, it should be "Using Serverless Dashboard Providers with IAM Role". I also think we should update all the references from "Serverless Providers" to "Serverless Dashboard Providers" to not confuse with the `provider.profile` and `provider.resolver` configurations.